modules/account-lib/test/unit/coin/algo/utils.ts

```diff
@@ -335,4 +336,19 @@ describe('utils', () => {
     const size = Algo.algoUtils.getTransactionByteSize(keyregTxn);
     should.equal(size, 320);
   });
+
+  it('Should be able to get a txID from a multising Tx', () => {
```

Comment:
Please try to do a failure case like an a not multisig tx, the enconding object for multisig should be fail, just to be clear that this txId is only for multisig tx to bring support for double spend.